src/routes/people/+page.svelte

```diff
@@ -12,25 +16,89 @@
-        <Exec src={beaDiokno} alt="Bea Diokno" pos="Director for External Relations" />
+<section>
+    <section class="prose mb-8 flex max-w-none flex-col gap-8">
+        <figure class="m-0 flex flex-col items-center">
+            <!-- <picture class="max-w-3xl">
+                <source type="image/webp" srcset={execSmall} media="(max-width:640px)" />
+                <source type="image/webp" srcset={execMedium} />
+                <img src={execFull} width="1568" height="1203" alt="The Executive Board" />
+            </picture> -->
+            <figcaption>The 2024-2025 Executive Board</figcaption>
+        </figure>
+        <section class="not-prose flex flex-wrap justify-around gap-8">
+            <Exec src={lino} alt="Andri Crisostomo" pos="President" />
+            <Exec src={lino} alt="Andri Crisostomo" pos="Vice President for Innovation" />
+            <Exec src={lino} alt="David Zamuco" pos="Vice President for Service" />
+            <Exec src={lino} alt="Victor Reyes" pos="Director for Engineering" />
+            <Exec src={lino} alt="Bianca Amurao" pos="Director for Membership & Internals" />
+            <Exec src={lino} alt="Clarisse Bucu" pos="Director for Branding & Creatives" />
+            <Exec src={lino} alt="Sean Tolentino" pos="Director for External Relations" />
+        </section>
     </section>
     <TeamPanel />
-</article>
+</section>
+<section>
+    <div class="mb-4 flex w-full flex-col items-center text-center">
+        <h1 class="w-full text-3xl text-csi-black sm:w-3/4 md:text-4xl dark:text-csi-white">
+            The Past Executive Boards
+        </h1>
+    </div>
+
+    <button
+        class="border-inherit mb-4 w-full border-b-2 text-csi-black dark:text-csi-white"
+        on:click={() => {
+            pastExec = !pastExec;
+        }}
+    >
+        <div class="flex w-full flex-row items-center text-center">
+            {#if pastExec}
+                <img
+                    src={$theme ? arrowUpLight : arrowUpDark}
+                    alt="Toggle Dropdown"
+                    class="h-4 pr-2"
+                />
+            {:else}
+                <img
+                    src={$theme ? arrowDownLight : arrowDownDark}
+                    alt="Toggle Dropdown"
+                    class="h-4 pr-2"
+                />
+            {/if}
+            <h2 class="text-xl md:text-2xl">2023-2024</h2>
+        </div>
+    </button>
+    {#if pastExec}
+        <section class="prose flex max-w-none flex-col gap-8">
+            <figure class="m-0 flex flex-col items-center">
+                <picture class="max-w-3xl">
+                    <source type="image/webp" srcset={execSmall} media="(max-width:640px)" />
+                    <source type="image/webp" srcset={execMedium} />
+                    <img src={execFull} width="1568" height="1203" alt="The Executive Board" />
+                </picture>
```

Comment:
Maybe we should consider using [`@sveltejs/enhanced-img`](https://kit.svelte.dev/docs/images#sveltejs-enhanced-img) here. As mentioned in my InnovCamp 2024 talk, the `<enhanced:img>` helper automatically expands to this `<picture>` element anyway.

Reply:
For the sake of terseness, let's take this on too @galierie .

Reply:
Hello! I'm running into an issue using `<enhanced:img>`. The `src` attribute doesn't seem to read the path whenever I import a picture from `$lib/people`. I had to put the relative file path directly into the `src` attribute so it'd work. This shouldn't be the case, no?

Reply:
Yeah, according to the docs, any Vite import path should be allowed.

```html
<enhanced:img src="$lib/people/.../yenyen.webp" />
```

Reply:
Testing it locally and based on [this issue](https://github.com/sveltejs/kit/issues/11535), it should also work if the `?enhanced`  query parameter is added to the import:

```svelte
import exec2324 from '$lib/people/exec.webp?enhanced';
```

(Sidenote, but is it just me, or is [this particular part](https://kit.svelte.dev/docs/images#sveltejs-enhanced-img-dynamically-choosing-an-image) of the Svelte documentation rather confusingly written?)

Reply:
Yes, that's fair enough. 😅

With Hacktoberfest coming up soon, would you like to submit a PR to them? 👀

Reply:
Ah maybe I should. Though, I'll think about it once October starts. 😄

Reply:
It worked yay! Tytysm!